src/news/release-5.1.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Prism Launcher Release 5.1, now available
+description: Too Many Fixes
+date: 2022-11-01
+release_version: 5.1
+minimum_macos_version: 10.14.0
+mac_signature: pineapple
+tags:
+  - release
+---
+
+Hi there!
+
+We are bringing you Prism Launcher 5.1!
+This is our first patch release after the *thing*.
+While we promised to bring you a blog article about the *thing*, it seems like interest has died down and there are many good write-ups/explanations of the situation on the internet.
+We did update [our FAQ] though, so feel free to read that.
+
+Now to the update.
+This mainly consists of bug fixes, nothing too exciting.
+We are already hyped for our next feature release though, which will bring some UI improvements.
+
+## Changelog
+
+### Added
+
+- Add support for *certain* languages by @Scrumplex in #354
+- Remove duplicate Java entries from Auto-detect by @AliceDTRH in #234
```

Comment:
Isn't this technically a fix?

Reply:
put it to changes instead. It wasn't really any breakage. It was just confusing to users.